cgd_tx_eff/src/edu/ohsu/compbio/annovar/annovar_parser.py

```diff
@@ -89,37 +91,7 @@ class AnnovarParser(object):
     Parses Annovar output
     ''' 
     def __init__(self):
-        self.hgvs_parser = hgvs.parser.Parser()
-    
-    def _get_file_type(self, file_name:str):
```

Comment:
I was originally determining the annovar file type (annovar.exonic_variant_function or annovar.variant_function) by its extension, but it then realised that Galaxy renames everything and gives it a .dat extension.

So then I tried to analyse the file to determine the file type. 

Now you just tell the tool explicitly which is which:
time python $TFX_HOME/src/edu/ohsu/compbio/txeff/tx_eff_control.py \
       --require_match \
       --in_vcf in.vcf \
       --ccds_map /opt/transcript_effects/ncbi/GRCh37_latest_genomic.csv \
       --out_vcf out.vcf \
       --annovar_variant_function annovar.variant_function \
       --annovar_exonic_variant_function annovar.exonic_variant_function > tx_eff.log 2>&1